### ISSUE TITLE
training error

### ISSUE THREAD

**Issue by JGyoung33** [chaoyuaw/pytorch-coviar#24]
Hi, I met with several problems in the training process.

ERROR: Unexpected segmentation fault encountered in worker.
ERROR: Unexpected segmentation fault encountered in worker.
[jpegls @ 0x562d3c3db340] Found EOI before any SOF, ignoring
[jpegls @ 0x562d3c3db340] unable to decode APP fields: Invalid data found when processing input
[jpegls @ 0x562d3c3db340] unable to decode APP fields: Invalid data found when processing input
ERROR: Unexpected segmentation fault encountered in worker.
ERROR: Unexpected segmentation fault encountered in worker.
[jpegls @ 0x562d3c3db340] mjpeg: unsupported coding type (c5)
[jpegls @ 0x562d3c3db340] dqt: invalid precision
Decode Error.
Decoding video failed.
Error: loading video data/hmdb51/mpeg4_videos/stand/Man_Who_Cheated_Himself_512kb_stand_u_cm_np1_fr_med_11.mp4 failed.

**Comment by chaoyuaw** [comment]
Hi @JGyoung33 , it looks like ffmpeg failed to load unfortunately. Were you able to run reencode.sh successfully? Were you able to use the same version of FFmpeg as what's described in GETTING_STARTED.md?

**Comment by JGyoung33** [comment]
> Hi @JGyoung33 , it looks like ffmpeg failed to load unfortunately. Were you able to run reencode.sh successfully? Were you able to use the same version of FFmpeg as what's described in GETTING_STARTED.md?

Yes ,the ffmpeg version is same and I run reencode.sh successfully. This error sometimes would appear randomly, for example, next time it would fail to decode another video. 

I think I have another important error, and every time I run my program, there is an error below:
"Cannot open input stream"

![125084375038619931](https://user-images.githubusercontent.com/34210827/46985763-9f802680-d11e-11e8-9ad8-d7a0182feaf4.jpg)

How should I solve it? Thank you for replying.

**Comment by JGyoung33** [comment]
> > Hi @JGyoung33 , it looks like ffmpeg failed to load unfortunately. Were you able to run reencode.sh successfully? Were you able to use the same version of FFmpeg as what's described in GETTING_STARTED.md?
> 
> Yes ,the ffmpeg version is same and I run reencode.sh successfully. This error sometimes would appear randomly, for example, next time it would fail to decode another video.
> 
> I think I have another important error, and every time I run my program, there is an error below:
> "Could not open input stream"
> 
> ![125084375038619931](https://user-images.githubusercontent.com/34210827/46985763-9f802680-d11e-11e8-9ad8-d7a0182feaf4.jpg)
> 
> How should I solve it? Thank you for replying.

**Comment by filipetrocadoferreira** [comment]
I have the same problem

**Comment by SmartLiam** [comment]
> I have the same problem

did you solve the problem

**Comment by Maruidear** [comment]
> > 我也有同样的问题
> 
> 你解决问题了吗

Have you solved the problem?
